src/inspect_ai/model/_call_tools.py

```diff
@@ -96,6 +100,17 @@ async def call_tool_task(call: ToolCall) -> tuple[ChatMessageTool, ToolEvent]:
                 if isinstance(ex.filename, str):
                     err = f"{err} Filename '{ex.filename}'."
                 tool_error = ToolCallError("is_a_directory", err)
+            except SandboxOutputLimitExceededError as ex:
+                tool_error = ToolCallError(
+                    "output_limit",
+                    f"The sandbox exec output limit of {ex.limit_str} was exceeded.",
+                )
+                result = ex.truncated_result
```

Comment:
How this manifests itself to the model:
![Screenshot 2024-10-31 at 09 25 03](https://github.com/user-attachments/assets/100a76f5-5e82-4441-8a1d-ad6431aa057b)